src/types.ts

```diff
@@ -221,3 +221,22 @@ export type Voucher = {
   serial: string;
   denomination: number;
 };
+
+export const Statistics = t.intersection([
+  t.type({
+    category: t.string,
+    quantity: t.number,
+    transactionTime: t.number
+  }),
+  t.partial({
+    identifierInputs: t.array(IdentifierInput)
+  })
+]);
+
+export type Statistics = t.TypeOf<typeof Statistics>;
```

Comment:
Is this required too? Since we only use it for  `DailyStatistics`, I think we don't need to convert it to a TypeScript type 🤔

Reply:
removed!